gcp/api/server.py

```diff
@@ -35,14 +38,13 @@
 import osv_service_v1_pb2
 import osv_service_v1_pb2_grpc
 
-from typing import List
-
 _SHUTDOWN_GRACE_DURATION = 5
 
 _MAX_BATCH_QUERY = 1000
 _MAX_VULNERABILITIES_LISTED = 16
-_MAX_HASHES_TO_TRY = 50
-_MAX_COMMITS_TO_TRY = 10
+_MAX_MATCHES_TO_CARE = 100
```

Comment:
Can you add more comments here to describe these constants? 

Some of these (e.g. `_MAX_RESULTS_TO_RETURN` ) also sound a bit generic when they're only applicable ot the determine versions endpoint.